### DISCUSSION TITLE
Truth information and ckf barcode

### DISCUSSION THREAD

**Discussion by rubihason** [acts-project/acts#2075] [Q&A]
Hello, i want to ask if the information about the particle from the barcode is it Truth information. If we take this barcode for example:
1|1|48|0|0
Does this say that this particle came for the hard scatter based on pythia8 information? Or is based on Ckf algorithm (reconstructed information).

**Comment by andiwand** [comment]
hey @rubihason!

the barcode is purely simulation as far as I know and the primary vertex id should indeed number the one created with pythia.

hope this helps

**Reply by rubihason** [reply]
Yes, it was.
Thank you!
